libraries/BLE/examples/central/central.ino

```diff
@@ -0,0 +1,94 @@
+
+#include "ArduinoBLE.h"
+#include "BLEAttribute.h"
+#include "BLECharacteristicImp.h"
+#include "BLEProfileManager.h"
+
+// LED pin
+#define LED_PIN   13
+
+void setup() {
+    Serial.begin(115200);
+    Serial.println("test---");
+    
+    // set LED pin to output mode
+    pinMode(LED_PIN, OUTPUT);
+
+    // begin initialization
+    BLE.begin();
+    Serial.println(BLE.address());
+    
+    BLE.startScanning("LED");

```

Comment:
I've started another discussion on this outside this PR.

We can drop discovered peripherals if the user does not call `available()` fast enough.

We should not expose the raw advertisement and scan response to the user. There are some existing API's in BLE devices for them to access the parsed data.